exposed-version-catalog/build.gradle.kts

```diff
@@ -0,0 +1,95 @@
+plugins {
+    `version-catalog`
+    `maven-publish`
+    signing
+}
+
+group = "org.jetbrains.exposed"
+version = rootProject.version
+
+catalog {
+    versionCatalog {
+        // Core modules
```

Comment:
@jetbrains-junie this should be wrapped in `create("exposed") { }` to achieve `exposed.core` naming scheme.